File: Java/Storage.java
public class Storage {
    private int kapasitasGB;
    private String tipeDrive;
    private String merk;
    private String nama;

    public Storage(int kapasitasGB, String tipeDrive, String merk, String nama) {
        this.kapasitasGB = kapasitasGB;
        this.tipeDrive = tipeDrive;
        this.merk = merk;
        this.nama = nama;
    }

    public int getKapasitasGB() { return kapasitasGB; }
    public String getTipeDrive() { return tipeDrive; }
    public String getMerk() { return merk; }
    public String getNama() { return nama; }
}

File: CPP/Storage.cpp
#pragma once
#include <iostream>
#include "Komponen.cpp"

using namespace std;

class Storage : public Komponen {
private:
    int kapasitasGB;
    string tipeDrive;
public:
    Storage() {}

    Storage(int kapasitasGB, string tipeDrive, string merk, string nama)
        : Komponen(merk, nama) {
        this->kapasitasGB = kapasitasGB;
        this->tipeDrive = tipeDrive;
    }

    void setKapasitasGB(int kapasitasGB) { this->kapasitasGB = kapasitasGB; }
    int getKapasitasGB() { return kapasitasGB; }
    string getTipeDrive() { return tipeDrive; }

    ~Storage() {}
};
